lib/services/database_test.go

```diff
@@ -189,3 +189,20 @@ func TestDatabaseFromRDSCluster(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, expected, actual)
 }
+
+func TestDatabaseFromRDSCluster_serverless(t *testing.T) {
```

Comment:
Nit: We don't usually separate test names with underscores.

```suggestion
func TestDatabaseFromRDSClusterServerless(t *testing.T) {
```

You run use  `t.Run()` for named "sub-tests".